packages/fluentui/react-builder/src/hooks/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './useAxeOnElement';
+export * from './useMode';
```

Comment:
Do we need these `index` reexports? I usually prefer to import a single thing from a single file.

Reply:
Well, this was we can do like this:

`import { HookX, HookY } from './hooks`

if we remove the index we would have more lines in the imports to match this:

```
import { HookX } from './hooks/HookX
import { HookY } from './hooks/HookY
```